.github/workflows/upgrade-python-requirements.yml

```diff
@@ -0,0 +1,71 @@
+name: Upgrade Requirements
+
+on:
+  push:
+    branches:
+      - '**'
+  schedule:
+    # will start the job at 3:15 UTC every Thursday
+    - cron: "15 3 * * 4"
```

Comment:
Could you schedule this once every other week ([as before](https://github.com/edx/jenkins-job-dsl/blob/32b331a9c1a3b2683e3bd7e191062cc72b0f1094/testeng/jobs/upgradePythonRequirements.groovy#L273)) instead? Additionally it would be helpful if it wasn't always on Thursday